Content/Pferderennen.tex

```diff
@@ -2,14 +2,14 @@ \chapter{Pferderennen}
 \section{Vorbereitung}
 \begin{enumerate}[label={(\arabic*)}]    
     \item
-    Die Asse eines 32er Kartensets werden zur Seite gelegt und die übrigen Karten gut gemischt.
+    Die Asse eines 52er Kartensets werden zur Seite gelegt und die übrigen Karten gut gemischt.
     Sieben zufällige Karten werden verdeckt in einer Reihe auf den Tisch gelegt.
     Die Asse werden offen ebenfalls in einer Reihe auf den Tisch gelegt, sodass sie orthogonal zur ersten Reihe kurz vor Beginn der vorherigen Reihe liegen.
```

Comment:
```
+---+---+----+----+----+----+----+----+
|   | X |  X |  X |  X |  X |  X |  X |
| A |   |    |    |    |    |    |    |
| A |   |    |    |    |    |    |    |
| A |   |    |    |    |    |    |    |
| A |   |    |    |    |    |    |    |
+---+---+----+----+----+----+----+----+

A = Ass
X = verdekte zufällige Karte
```
^- könnte man als Visualisierung hinzufügen, ansonsten finde ich die Beschreibung etwas schwer zu verstehen, vermutlich weil ich mit `Reihe` immer direkt an eine Tabelle denken muss, was dann eine orthogonale `Reihe` etwas verwirrend ist (quasi eine Spalte).

Reply:
Fair. Ich hab bisher versucht Visualisierungen zu vermeiden. Sind anfällig irgendwas zu zerschießen und nicht barrierefrei. Stattdessen eine Idee wie man den Text eindeutiger gestalten kann?

Reply:
Würde einfach schreiben, dass man die Asse in eine Spalte anordnet?

Reply:
check mal die aktuelle version ab

Reply:
> Fair. Ich hab bisher versucht Visualisierungen zu vermeiden. Sind anfällig irgendwas zu zerschießen und nicht barrierefrei.

Inwiefern ist eine Text basierte Visualisierung weniger barrierefrei als Text? wegen text-to-speech?

Quick & dirty tex code:
```TeX
\begin{verbatim}
+---+---+----+----+----+----+----+----+
|   | X |  X |  X |  X |  X |  X |  X |
| A |   |    |    |    |    |    |    |
| A |   |    |    |    |    |    |    |
| A |   |    |    |    |    |    |    |
| A |   |    |    |    |    |    |    |
+---+---+----+----+----+----+----+----+

A = Ass
X = verdekte zufällige Karte
\end{verbatim}
```

> Würde einfach schreiben, dass man die Asse in eine Spalte anordnet?

Finde ich persönlich verständlicher, ja.

Reply:
zu liebevoll um es nicht zu verwenden ^^ ist drin

Reply:
> Inwiefern ist eine Text basierte Visualisierung weniger barrierefrei als Text? wegen text-to-speech?

Ja genau, aber spricht ja nichts dagegen zu einer möglichst guten Beschreibung eine Grafik einzufügen